app/views/custom_styles/show.html.erb

```diff
@@ -77,17 +77,32 @@ See doc/COPYRIGHT.rdoc for more details.
           <label class="form--label"><%= design_color.variable %>:</label>
           <span class="form--field-container">
             <span class="form--text-field-container">
-              <%= styled_text_field_tag "design_colors[]" + design_color.variable, design_color.hexcode, placeholder: DesignColor.defaults[design_color.variable] %>
+              <%= styled_text_field_tag "design_colors[]" + design_color.variable,
+                                        design_color.hexcode,
+                                        placeholder: (DesignColor.defaults[design_color.variable] =~ /#[0-9A-F]{6}/ ? DesignColor.defaults[design_color.variable] : '') %>
             </span>
           </span>
-          <%= icon_for_color(OpenStruct.new( variable: design_color.variable, hexcode: design_color.get_hexcode ), class: 'timelines-x-update-color timelines-color-properties-preview', data: { target: '#color_hexcode' }) %>
+          <%= icon_for_color OpenStruct.new( variable: design_color.variable,
+                                             hexcode: design_color.get_hexcode ),
+                             class: 'timelines-x-update-color timelines-color-properties-preview',
+                             data:  { target: '#color_hexcode' } %>
           <div class="form--field-instructions">
             <% if design_color.variable == "primary-color" %>
               <%= t("admin.custom_styles.instructions.primary_color") %>
             <% elsif design_color.variable == "primary-color-dark" %>
               <%= t("admin.custom_styles.instructions.primary_color_dark") %>
             <% elsif design_color.variable == "alternative-color" %>
               <%= t("admin.custom_styles.instructions.alternative_color") %>
+            <% elsif design_color.variable == "header-bg-color" %>
```

Comment:
This is starting to get out of hand. What's wrong with `design_color.variable.tr('-', '_')` ? :-)